### PR TITLE
DBT2 benchmark execution against AWS BigAnimal PG

### PR DESCRIPTION
**Pull request by mw2q**
Environment variables that must be passed by buildbot:

TERRAFORM_PROJECT_PATH - Absolute path to the terraform project created by edb-terraform.
DBT2_DURATION - Duration in seconds of DBT2 execution. DBT2_WAREHOUSE - Number of warehouse.
DBT2_CONNECTIONS - Number of database connections.

### CODE REVIEW

File: aws-dbt2-ba/provision/run.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eux
+
+edb-terraform ${TERRAFORM_PROJECT_PATH} ../infrastructure.yml
+cd ${TERRAFORM_PROJECT_PATH}
+terraform init
+terraform apply -var-file=./terraform_vars.json -auto-approve
+
+biganimal create-cluster --cluster-config-file ../ba-infrastructure.yml -y
+# TODO: how to make script wait until bignimal finishes provisioning?
```
Comment (mw2q) [inline aws-dbt2-ba/provision/run.sh:9]: I think I need some help here...

Comment (jt-edb) [reply aws-dbt2-ba/provision/run.sh:9]: Wondering what would be the return of `biganimal show-clusters --id` if the cluster is not yet ready. If it returns an error, maybe you can create a loop and execute the `show-clusters` command until the cluster is ready?

Comment (mw2q) [reply aws-dbt2-ba/provision/run.sh:9]: Ok, I will experiment with that and report back.

**Comment by mw2q** [comment]
@jt-edb I think it's complete.  Ready for a review on the BigAnimal waiting.

File: aws-dbt2-ba/provision/run.sh (hunk 1 above)
Comment (mw2q) [reply aws-dbt2-ba/provision/run.sh]: Done: [e764e30](https://github.com/EnterpriseDB/edb-benchmarks/pull/7/commits/e764e301978abdf2e25cdf0cf1644d6abeb036d3)

File: aws-dbt2-ba/execute/playbook-dbt2-run.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            -C {{ infra.servers.machines["dbt2-client"].private_ip }}
+            -d {{ dbt2_duration }}
+            -D {{ infra.servers.databases.dbt2.dbname }}
+            -H {{ infra.servers.databases.dbt2.address }}
+            -n
+            -o /tmp/dbt2-data
+            -u
+            -w {{ dbt2_warehouse }}
+            -y
+      become: true
```
Comment (jt-edb) [inline aws-dbt2-ba/execute/playbook-dbt2-run.yml:39]: Please set the PGHOST environment variable here.

Comment (mw2q) [reply aws-dbt2-ba/execute/playbook-dbt2-run.yml:41]: Done. [864abd8](https://github.com/EnterpriseDB/edb-benchmarks/pull/7/commits/864abd8814af51bd6dd512a9f314ee2bb4aee40c)

**Review by jt-edb — APPROVED** [review]
LGTM

**Pull request merged**